src/logion/controllers/locrequest.controller.ts

```diff
@@ -510,7 +537,7 @@ export class LocRequestController extends ApiController {
     @SendsResponse()
     async downloadFile(_body: any, requestId: string, hash: string): Promise<void> {
         const request = requireDefined(await this.locRequestRepository.findById(requestId));
-        const contributor = await this.ensureContributor(request);
+        const contributor = await this.ensureContributorOrVoter(request);
 
         const file = request.getFile(hash);
         if(contributor !== request.ownerAddress && contributor !== request.requesterAddress && file.submitter !== contributor) {
```

Comment:
This condition should change as well, otherwise voters will not be able to download the file.